code/game/objects/items/weapons/melee/misc.dm

```diff
@@ -21,9 +21,9 @@
 		return (OXYLOSS)
 
 
-/obj/item/weapon/melee/rapier
-	name = "captain's rapier"
-	desc = "An elegant weapon, for a more civilized age."
+/obj/item/weapon/melee/sabre
+	name = "officer's sabre"
+	desc = "An elegant weapon, it's monomolecular edge is capable of cutting through flesh and bone with ease."
```

Comment:
The issue was people thinking of rapiers as pointy swords, naming it something else already accomplishes that